website/content/partials/known-issues/1_16-default-lcq-pre-1_9-upgrade.mdx

```diff
@@ -0,0 +1,26 @@
+### Default Lease Count Quota enabled when upgrading from Vault versions before 1.9
+
+#### Affected versions
+
+- 1.16.0
```

Comment:
Hmm, this would affect every version from 1.16 onwards, right? So 1.16.1, 1.16.2, 1.17.5, etc. I'm not sure how we usually do this, but it'd be great if we could indicate as much in the 'affected versions'.

Reply:
I think we instruct customers to read breaking changes in previous versions, too, so I don't know that there's a worry about this not being in 1.17, but I want to avoid folks reading it in a way that sounds like it might be 'fixed' in the future

Reply:
We can use 1.16+ as the affected version to indicate that it affects every version starting with 1.16

Reply:
Good call. We can just say 1.16.0+, I think!